src/main/java/org/alliancegenome/curation_api/model/entities/orthology/GeneToGeneOrthologyCurated.java

```diff
@@ -24,6 +25,7 @@
 import lombok.ToString;
 
 @Audited
+@Indexed
```

Comment:
@markquintontulloch is there going to be a table to view all Gene to Gene ortho? This may cause more indexing issues. To me this would be another candidate for a gene page that has a table to show them?

Reply:
@oblodgett  - UI code and @Indexed annotations removed